.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and publish AppImages
+
+env:
+  UBUNTU_RELEASE: jammy
+  UBUNTU_PUBKEY: 871920D1991BC93C
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v2
```

Comment:
Maybe skip if platform is `linux/amd64`?

Reply:
Hmm, I don't know. That imposes a dependency on the platform of the underlying runner. Currently it's `amd64` for sure, but I wouldn't put that knowledge into the workflow. The benefit will be small, I guess.

Reply:
GitHub won't be switching builders to a different host architecture overnight; it's a perfectly safe thing to depend on (and a trivial change if it ever stops being such). In a sense, every workflow that does not bother with docker+qemu yet produces binaries relies on this.

Reply:
Done.